Misc/NEWS.d/next/Build/2020-05-25-09-46-32.bpo-40747.ef15ix.rst

```diff
@@ -0,0 +1 @@
+The PEP 425 python tag, taken from ``py_version_nodot``, adds a ``_`` so ``cp310`` is now ``cp3_10``.
```

Comment:
```suggestion
The PEP 425 python tag, taken from ``py_version_nodot``, adds a ``_`` so ``cp310`` is now ``cp3_10``.
Patch by Matti Picus.
```

Reply:
None of the other blurbs in that directory have a contributor name.

Reply:
I would prefer not to, but won't stop you from commiting the change.